wizeline/remix-project-lab
# Feat/108/Project membership Modal

# What does this PR do?
We'll show a modal when the user has some days without interaction in the page in order to update their status in the projects.



# Where should the reviewer start?
To show the modal, we take as reference the updatedAt field of projectMembers table as the last date of interaction of an user.



# How should this be manually tested?

1. Join a project
2. Look for your user in the projectMembers table and change the updatedAt field.
3. Go to the /projects path
4. Update the fields related to the project
5. Check that the modal doesn’t appear again.



# Any background context you want to provide?



# What are the relevant tickets?
#108 



# Screenshots
![image](https://github.com/wizeline/remix-project-lab/assets/74632242/66da2eb4-79cc-408d-a1ba-18a5b98057b7)
![image](https://github.com/wizeline/remix-project-lab/assets/74632242/62b555f0-6a18-431b-b58a-57fa9c0814b7)
![image](https://github.com/wizeline/remix-project-lab/assets/74632242/a56d7103-ee36-470e-981c-6c6dbab73ba0)



# Questions



# Checklist
